public/index.html

```diff
@@ -6,6 +6,6 @@
     <title>Static list of posts</title>
   </head>
   <body>
-    <div id="root"></div>
+    <div id="root" class="wrapper"></div>
```

Comment:
```suggestion
    <div id="root"></div>
```
При роботі з реактом ми не додаємо ніяких класів для кореневого елементу.
Всі класи потрібно задавати в самому реакті